sycl/doc/developer/ContributeToDPCPP.md

```diff
@@ -62,6 +62,8 @@ end-to-end or SYCL-CTS tests.
 - Add a helpful comment describing what the test does at the beginning and
   other comments throughout the test as necessary.
 
+- All identifiers used in `llvm/sycl` headers and source files must contain at least one lowercase letter due to avoid conflicts with user-defined macros. 
```

Comment:
Please, limit the line length to 80 characters.

Reply:
Sure.